packages/@webex/webex-core/src/lib/credentials/credentials.js

```diff
@@ -528,9 +548,7 @@ const Credentials = WebexPlugin.extend({
         this.scheduleRefresh(this.supertoken.expires);
       })
       .catch((error) => {
-        const {InvalidRequestError} = grantErrors;
-
-        if (error instanceof InvalidRequestError) {
+        if (error instanceof OAuthError) {
```

Comment:
I am kind of worried about this tbh because we have no idea if this is happening or why. Incidentally, Token refresh function catches processGrantError

Reply:
I moved the error handling closer to the super token refresh, so we can be sure that failed and not a user token fetch.

Regarding the error check, we can not do anything except logout the user when the super token refresh fails